package.json

```diff
@@ -46,10 +46,9 @@
   "dependencies": {
     "bluebird": "^2.9.32",
     "ext2fs": "0.0.8",
-    "fatfs": "^0.10.3",
+    "fatfs": "https://github.com/zvin/fatfs/tarball/error-event-if-init-fails",
```

Comment:
There's even a shorthand for GitHub repos (@lurch For arbitrary git repos you'd want to use `git+https://` instead of `git://` though):

```js
// GitHub
"fatfs": "github:zvin/fatfs#error-event-if-init-fails"
// Git
"fatfs": "git+https://github.com/zvin/fatfs.git#error-event-if-init-fails"
```

But tarballs work just as well